### ISSUE TITLE
Zodiac Constellation

### ISSUE THREAD

**Issue by Zemelia** [cosinekitty/astronomy#366]

**Comment by cosinekitty** [comment]
Hi @Zemelia! What you are seeing is due to the Moon sometimes moving significantly north or south of the ecliptic plane, away from where all the zodiac constellations are located. 

Ancient astrologers tended to ignore ecliptic latitudes of the Moon and planets. They would focus on ecliptic longitudes, as if projecting their shadows onto a plane. 

They would often divide the 360 degrees of longitude into 12 equal bands of 30 degrees, which differs from the modern (1875 and after) constellation boundaries defined by the IAU; these boundaries are of all different sizes and shapes, not at all equal. 

You may have better luck calculating ecliptic coordinates instead of equatorial, and taking a look at the longitudes thus obtained.

**Comment by Zemelia** [comment]
Hi @cosinekitty , first of all thank you for your great and titan work on this library, it's awesome! And thanks for answer :)
Acc. your answer, in context of ecliptic coordinates, so how could I use `Constellation` function as there I need `ra` and `dec` args?

**Comment by cosinekitty** [comment]
Yes, your code example implements the idea I posed. Whether this idea is "correct" or useful is another matter; astrology isn't believable to me so I don't offer this as being "right" in any sense.
